hosted/aks/p1/p1_provisioning_test.go

```diff
@@ -184,6 +323,37 @@ var _ = Describe("P1Provisioning", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("should not be able to edit availability zone of a nodepool", func() {
```

Comment:
Should this be marked XIt ?
Since the current assertion is not the expected result.

Reply:
User is not allowed to edit the availability zone, even though POST request does not return an error, the state of AZ is restored. Hence, I did not mark the test as XIt.

Reply:
Ok, can we add check to find the AZ are indeed restored.

Reply:
It has been added on L#345-354. I could also add UpstreamSpec check if that is what you meant.